test/WebSites/ModelBindingWebSite/Models/Resident.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc;
+
+namespace ModelBindingWebSite
+{
+    public class Resident : Person
+    {
+        public IEnumerable<Address> ShippingAddresses { get; set; }
+
+        public Address HomeAddress { get; set; }
+
+        [FromBody]
+        public Address OfficeAddress { get; set; }
```

Comment:
Can we get scenarios with `[FromServices]` in here to ensure they don't get validated? 

```C#
[FromServices]
public TestService Service
{
   get { throw new Exception("Shouldn't be called"}; }
   set { }
}
```

Also we should figure out what it means to validate things that previously wouldn't be validated - `CancellationToken`, `IFormCollection`, result of byte array model binder etc